CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing
+
+This page lists the steps needed to set up a development environment and contribute to the project.
+
+1. Fork and clone this repo
+
+2. Install initial dependencies in a virtual environment:
+
+   ```shell
+   python -m venv venv
+   source venv/bin/activate
+   python -m pip install --upgrade pip 'tox>=4.2'
```

Comment:
Just link to https://tox.wiki/en/4.11.3/installation.html#via-pipx, don't duplicate documentation.